docs/rest.md

```diff
@@ -593,4 +593,59 @@ A new file with the following naming convention
 ```
 curl --location --request POST 'http://localhost:8180/api/v2/ema/resources/import' \
 --form 'file=@"{full path to the .zip file}'
-```
\ No newline at end of file
+```
+
+## Manual Import of Off-Line Agent Discovery
+The results of a scan run against one messaging service may be uploaded for audit and import to another messaging service defined in Event Portal. 
+
+### Procedure
+In essence, you will perform a scan from one agent, export it, and upload it to Event Portal using another agent. However, the exported data contains identifiers specific to the agent that performed the scan. These identifiers must be modified to suit the agent performing the upload, and modified to the correct messaging service.
+
+### Requirements
+- Source Configuration (off-line):
+    - EMA configuration (e.g. `your-source-config.yml`)
+    - Stand-alone setting path: `eventPortal.gateway.messaging.standAlone: true`
```

Comment:
please change `standAlone` to `standalone` to match the config name

Reply:
done